validator/testcases/javascript/instanceproperties.py

```diff
@@ -225,6 +252,8 @@ def set_contentScript(value, traverser):
                         "is dangerous and error-prone. Please use a separate "
                         "JavaScript file, along with the "
                         "`contentScriptFile` property instead.",
+            signing_help="Please do not use the `contentScript` property "
+                         "in any add-ons submitted for automated signing.",
```

Comment:
So, maybe it should state that in the message? I personally did not know there was a difference in the review between listed and unlisted add-ons (apart from the "policy" restrictions). So, just to clarify: `contentScript` will never be allowed in unlisted add-ons? Or maybe only in side-loaded add-ons?